src/components/badges/List.tsx

```diff
@@ -54,33 +55,37 @@ function BadgeListSuspended() {
   ) : isEmpty ? (
     <HintCard text="You don't own any derivatives and you don't have any ZK proofs ready to use. Generate a ZK proof first!" />
   ) : (
-    <Scrollbar>
-      <div className={space('space-y-2')}>
-        <BadgeSection
-          title="Mainnet NFT derivatives"
-          minted={ownedExternalERC721DerivativeContracts}
-          proofs={proofsAvailableToMint.filter(
-            (proof) =>
-              proof instanceof ERC721Proof && proof.network === Network.Mainnet
-          )}
-        />
-        <BadgeSection
-          title="Goerli NFT derivatives"
-          minted={ownedERC721DerivativeContracts}
-          proofs={proofsAvailableToMint.filter(
-            (proof) =>
-              proof instanceof ERC721Proof && proof.network === Network.Goerli
-          )}
-        />
-        <BadgeSection
-          title="Email derivatives"
-          minted={ownedEmailDerivativeContracts}
-          proofs={proofsAvailableToMint.filter(
-            (proof) => proof instanceof EmailProof
-          )}
-        />
-      </div>
-    </Scrollbar>
+    <>
+      <ConfettiIfNeeded />
```

Comment:
I would like to place it inside the `Scrollbar`, but it will cut the canvas because we have `overflow-x-hidden` inside the scrollbar